EnglishFilter/sections/general_extensions.txt

```diff
@@ -145,6 +145,9 @@ gotporn.com,porncomix.info,imgdew.com,webcamrips.com,webcamshows.org,mycumx.com,
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/39254
 protopage.com#%#AG_setConstant('displayAds', 'false');
+! https://github.com/AdguardTeam/AdguardFilters/issues/39123
+mangareader.net#%#AG_abortOnPropertyRead('document.querySelector', 'parts');
+mangareader.net#%#//scriptlet("abort-on-property-read", "parts", "document.querySelector")
```

Comment:
Shouldn't it be `AG_abortInlineScript/abort-current-inline-script` instead of  `AG_abortOnPropertyRead/abort-on-property-read`?

`mangareader.net#%#//scriptlet("abort-current-inline-script", "document.querySelector", "parts")`
`mangareader.net#%#AG_abortInlineScript(/parts/, 'document.querySelector');`

If not then I think that `AG_abortOnPropertyRead/abort-on-property-read` requires only one parameter, so it should be something like this:
`mangareader.net#%#//scriptlet("abort-on-property-read", "parts")`
`mangareader.net#%#AG_abortOnPropertyRead('parts');`

Reply:
It works in both cases. I will leave one parameter in `AG_abortOnPropertyRead/abort-on-property-read`. Thanks!